### DISCUSSION TITLE
Implementation of a SwithNavigator

### DISCUSSION THREAD

**Discussion by alfonsosn** [nandorojo/solito#349] [Q&A]
Hello!

First of all, just want to say that I really appreciate the work that @nandorojo did in bringing libraries like Solito and Dripsy into our ecosystem. I believe that your contribution will continue to be enhanced by community support!

Now, I have an architecture question for navigation in Solito. As you know, the SwitchNavigator is how most react-native apps can determine what routes to present to an authenticated state. Will the same concept apply in Solito for NextJS apps? I didn't see it in the docs nor the showtime-frontend code.

Thanks!

**Comment by nandorojo** [comment]
Hey, check out my latest Next.js conf talk where I discuss auth! I'll add a page to the docs about it

**Comment by derek-primumco** [comment]
This talk? https://www.youtube.com/watch?v=H1gSWXA3qfw  "Fernando Rojo: Is React Native + Next.js production-ready?"

**Comment by nandorojo** [comment]
yep
